src/Games/NexusMods.Games.StardewValley/Diagnostics.cs

```diff
@@ -204,11 +204,60 @@ internal static partial class Diagnostics
 
 You can download the latest supported SMAPI version {LatestSMAPIVersion} for your game version
 {CurrentGameVersion} from {SMAPINexusModsLink}.
-""")
+"""
+        )
         .WithMessageData(messageBuilder => messageBuilder
             .AddValue<string>("LatestSMAPIVersion")
             .AddValue<string>("CurrentGameVersion")
             .AddValue<NamedLink>("SMAPINexusModsLink")
         )
         .Finish();
+
+    [DiagnosticTemplate]
+    [UsedImplicitly]
+    internal static IDiagnosticTemplate SMAPIVersionOlderThanMinimumAPIVersion = DiagnosticTemplateBuilder
+        .Start()
+        .WithId(new DiagnosticId(Source, number: 11))
+        .WithTitle("SMAPI Version newer than supported Mod")
+        .WithSeverity(DiagnosticSeverity.Warning)
+        .WithSummary("The minimum supported SMAPI version of {ModName} is {MinimumAPIVersion}")
+        .WithDetails("""
+Mod {Mod} requires the SMAPI version to be at least {MinimumAPIVersion}.
+The current SMAPI version is {CurrentSMAPIVersion}.
+
+You can solve this issue by either updating SMAPI or download an older version
+of the mod from {NexusModsLink}.
```

Comment:
I would add both the link to SMAPI and the link to the mod page. We have the system to do that, might as well leverage it